src/main/java/frc/robot/CTREConfigs.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot;
+
+
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoderConfiguration;
+import com.ctre.phoenix.sensors.SensorInitializationStrategy;
+import com.ctre.phoenix.sensors.SensorTimeBase;
+
+import frc.robot.subsystems.swerve.SwerveConfig;
+
+public final class CTREConfigs {
+  
+    public CANCoderConfiguration swerveCanCoderConfig;
+
+    public CTREConfigs(){
+     
+        swerveCanCoderConfig = new CANCoderConfiguration();
+
+        
+        /* Swerve CANCoder Configuration */
+        swerveCanCoderConfig.absoluteSensorRange = AbsoluteSensorRange.Unsigned_0_to_360;
+        swerveCanCoderConfig.sensorDirection = SwerveConfig.canCoderInvert;
```

Comment:
Check with Armaan on this one, and if it should be configurable.  They recently solved the Kraken Drivebase problem for Redux by messing with the CAN coder invert setting.  @ak020107

Reply:
This class is never used in my codebase, so I'm going to delete it. I'll double check with all my files to see if there are any others that aren't used and delete them as well.